ICAFIX/PostFix.sh

```diff
@@ -35,210 +35,268 @@
-	if [ -z "${p_StudyFolder}" ]; then
-		log_Err "Study Folder (--path= or --study-folder=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "Study Folder: ${p_StudyFolder}"
-	fi
+#display the parsed/default values
+opts_ShowValues
 
-	if [ -z "${p_Subject}" ]; then
-		log_Err "Subject ID (--subject=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "Subject ID: ${p_Subject}"
-	fi
+log_Check_Env_Var CARET7DIR
+log_Check_Env_Var FSLDIR
 
-	if [ -z "${p_fMRIName}" ]; then
-		log_Err "fMRI Name (--fmri-name=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "fMRI Name: ${p_fMRIName}"
-	fi
+ReuseHighPass=$(opts_StringToBool "$ReuseHighPass")
 
-	if [ -z "${p_HighPass}" ]; then
-		log_Err "High Pass: (--high-pass=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "High Pass: ${p_HighPass}"
-	fi
+# Show HCP pipelines version
+log_Msg "Showing HCP Pipelines version"
+"${HCPPIPEDIR}"/show_version --short
 
-	if [ -z "${p_TemplateSceneDualScreen}" ]; then
-		log_Err "Dual Screen Template Scene (--template-scene-dual-screen=)required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "Dual Screen Template Scene: ${p_TemplateSceneDualScreen}"
-	fi
+# Show wb_command version
+log_Msg "Showing wb_command version"
+${CARET7DIR}/wb_command -version
 
-	if [ -z "${p_TemplateSceneSingleScreen}" ]; then
-		log_Err "Single Screen Template Scene (--template-scene-single-screen=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "Single Screen Template Scene: ${p_TemplateSceneSingleScreen}"
-	fi
+# Show FSL version
+log_Msg "Showing FSL version"
+fsl_version_get fsl_ver
+log_Msg "FSL version: ${fsl_ver}"
 
-	if [ -z "${p_ReuseHighPass}" ]; then
-		log_Err "Reuse High Pass (--reuse-high-pass=<YES | NO>) required"
-		error_count=$(( error_count + 1 ))
-	else
-		log_Msg "Reuse High Pass: ${p_ReuseHighPass}"
-	fi
+# show_usage()
+# {
+# 	cat <<EOF
 
-	if [ -z "${p_MatlabRunMode}" ]; then
-		log_Err "MATLAB Run Mode (--matlab-run-mode=) required"
-		error_count=$(( error_count + 1 ))
-	else
-		case ${p_MatlabRunMode} in
-			0)
-				log_Msg "MATLAB Run Mode: ${p_MatlabRunMode}"
-				if [ -z "${MATLAB_COMPILER_RUNTIME}" ]; then
-					log_Err_Abort "To use MATLAB run mode: ${p_MatlabRunMode}, the MATLAB_COMPILER_RUNTIME environment variable must be set"
-				else
-					log_Msg "MATLAB_COMPILER_RUNTIME: ${MATLAB_COMPILER_RUNTIME}"
-				fi
-				;;
-			1 | 2)
-				log_Msg "MATLAB Run Mode: ${p_MatlabRunMode}"
-				;;
-			*)
-				log_Err_Abort "MATLAB Run Mode value must be 0, 1 or 2"
-				error_count=$(( error_count + 1 ))
-				;;
-		esac
-	fi
-
-	if [ ${error_count} -gt 0 ]; then
-		log_Err_Abort "For usage information, use --help"
-	fi
-}
+# ${g_script_name}: Creates Workbench scene files that can be used to visually review
+# the signal vs. noise classification generated by ICA+FIX.
+
+# Usage: ${g_script_name} PARAMETER...
+
+# PARAMETERs are [ ] = optional; < > = user supplied value
```

Comment:
Old usage, other obsolete code should be removed.

Reply:
Done